task/check.go

```diff
@@ -0,0 +1,56 @@
+package task
+
+import (
+	"fmt"
+	"log"
+	"time"
+)
+
+func Check(t *Task) error {
+	log.Println("проверка задания")
+
+	if t.Title == "" {
+		return fmt.Errorf("заголовок пуст")
+	}
+
+	if t.Date == "" {
+		t.Date = time.Now().Format("20060102")
+	}
+
+	validDate, err := time.Parse("20060102", t.Date)
+	if err != nil {
+		return fmt.Errorf("дата имеет формат, отличный от 20060102")
```

Comment:
Теряешь сообщение об ошибки, оно может иногда сильно помогать
```suggestion
		return fmt.Errorf("дата имеет формат, отличный от 20060102: %w", err)
```